services/app-web/src/common-code/healthPlanFormDataType/healthPlanFormData.ts

```diff
@@ -251,28 +251,18 @@ const generateRateName = (
     return rateName
 }
 
-const removeRateRelatedDocuments = (
+const convertRateToContractDocs = (
     documents: SubmissionDocument[]
 ): SubmissionDocument[] => {
-    const noRateDocs: SubmissionDocument[] = []
-    documents.forEach((document) => {
-        const ratesRelatedDocIndex =
-            document.documentCategories.indexOf('RATES_RELATED')
-        const ratesDocIndex = document.documentCategories.indexOf('RATES')
-        const includesContractDocs =
+    return documents.map((document) => ({
+        ...document,
+        //Convert rate supporting documents to contract supporting and contract to rate documents.
+        documentCategories:
             document.documentCategories.includes('CONTRACT_RELATED') ||
-            document.documentCategories.includes('CONTRACT')
-        if (includesContractDocs) {
-            if (ratesDocIndex !== -1) {
-                document.documentCategories.splice(ratesDocIndex, 1)
-            }
-            if (ratesRelatedDocIndex !== -1) {
-                document.documentCategories.splice(ratesRelatedDocIndex, 1)
-            }
-            noRateDocs.push(document)
-        }
-    })
-    return noRateDocs
+            document.documentCategories.includes('RATES_RELATED')
+                ? ['CONTRACT_RELATED']
+                : ['CONTRACT'],
```

Comment:
What  are we doing with rate certifications now - RATES docs? If I'm reading this right it looked like they are being converted into the CONTRACT type? I think we might want to run this by design if you haven't already

Reply:
Good point. We want to remove rate certifications if we are changing submissions types. I thinking to scope this function to only work with supporting documents, 'CONTRACT_RELATED, RATES_RELATED', and throwing errors when certification documents are detected.

Reply:
Removing rate certifications is done at submit/resubmit in `submitHealthPlanPackage.ts` resolver